hack/make-rules/verify.sh

```diff
@@ -234,10 +234,9 @@ ret=0
 modules=() # Pacify shellcheck
 kube::util::read-array modules < <(go list -f '{{.Dir}}' -m)
```

Comment:
I just realize this omits these two
```
./staging/src/k8s.io/code-generator/examples/hack
./staging/src/k8s.io/apiextensions-apiserver/examples/client-go/hack
```

do we need them?

Reply:
Oops, nice catch — what prompted me to do all this initially is `./staging/src/k8s.io/code-generator/examples/hack`, so yes, we do need them!